### ISSUE TITLE
Cache hit / miss math

### ISSUE THREAD

**Issue by daniellienert** [t3n/neos-debug#128]
The cache hit / miss sums are wrong most of the times.

Example:

![image](https://user-images.githubusercontent.com/642226/102830383-80aa0880-43e9-11eb-90f2-e7191995bcc9.png)

**Comment by Sebobo** [comment]
Yes, for the absolute sum we should use the information from the individual blocks. I fixed some bugs there recently and they should be correct now.

The current numbers are generated from the calls and results of `replaceCachePlaceholders` and `getCachedSegment`. The question is whether it's correct that we have less cache calls than cached segments or not and if it's correct we also have to show those numbers.
Did you dig deeper into that already?
